src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -135,10 +135,7 @@ CHIP_ERROR pychip_DeviceController_NewDeviceController(chip::Controller::DeviceC
     initParams.storageDelegate              = &sStorageDelegate;
     initParams.mDeviceAddressUpdateDelegate = &sDeviceAddressUpdateDelegate;
     initParams.pairingDelegate              = &sPairingDelegate;
-
-#if CHIP_ENABLE_INTERACTION_MODEL
-    initParams.imDelegate = &PythonInteractionModelDelegate::Instance();
-#endif
+    initParams.imDelegate                   = &PythonInteractionModelDelegate::Instance();
```

Comment:
Ok. I guess this change explains why you're passing `nullptr` instead of callbacks at some point.